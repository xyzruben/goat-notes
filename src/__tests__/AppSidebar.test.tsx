import React from 'react';
import { render } from '@testing-library/react';
import AppSidebar from '@/components/ui/AppSidebar';

describe('AppSidebar', () => {
  it('renders without crashing', () => {
    const { container } = render(<AppSidebar />);
    expect(container).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    const { container } = render(<AppSidebar />);
    expect(container).toMatchSnapshot();
  });
}); 