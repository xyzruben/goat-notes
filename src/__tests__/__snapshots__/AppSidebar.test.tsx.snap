// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AppSidebar matches snapshot 1`] = `<div />`;
